ansible/examples/builder.yml

```diff
@@ -44,6 +43,62 @@
       tags:
         always
 
+    - name: Stop unattended-upgrades to avoid conflicts
+      systemd:
+        name: unattended-upgrades
+        state: stopped
+      when: ansible_os_family == "Debian"
+      ignore_errors: yes
```

Comment:
It's not even supposed to be installed in the first place https://github.com/ceph/ceph-build/blob/main/ansible/examples/builder.yml#L459-L464 😭